src/ngMessages/messages.js

```diff
@@ -24,10 +24,16 @@ var jqLite = angular.element;
  * `ngMessage` and `ngMessageExp` directives.
  *
  * # Usage
- * The `ngMessages` directive listens on a key/value collection which is set on the ngMessages attribute.
- * Since the {@link ngModel ngModel} directive exposes an `$error` object, this error object can be
- * used with `ngMessages` to display control error messages in an easier way than with just regular angular
- * template directives.
+ * The `ngMessages` directive listens on a key/value collection which is set on the ngMessages attribute of a container element.
+ * There should be one such element for each field whose error messages are to be displayed.
+ * The container element should have a series of child elements which each correspond to a possible error message for that field.
```

Comment:
It's not only error messages, it can be any string/message. The most common use is simply for $error elements.